modules/rostests/apitests/shell32/IShellFolderViewCB.cpp

```diff
@@ -586,18 +588,73 @@ START_TEST(IShellFolderViewCB)
         IShellFolderViewCB* oldPtr;
 
         hr = folderView->SetCallback(NULL, &oldPtr);
-        ok_int(g_AddRef, 1);
-        ok_int(g_Release, 0);
+        switch(GetNTVersion())
```

Comment:
```suggestion
        switch (GetNTVersion())
```
and same for the three other instances in this file.